### PR TITLE
{lang}[system] Julia v1.6.0 (aarch64)

### PR DESCRIPTION
**Pull request by rungitta**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by verdurin** [comment]
@rungitta welcome to EasyBuild and thanks for the contribution. 
Are you able to upload a test report on `aarch64` hardware?

**Comment by rungitta** [comment]
I'd be glad to upload a test report. How do I do that? Can you point me to the
relevant documentation?

-- 
Axel Rosén, Head Engineer
Research Infrastructure Services Group, USIT
University of Oslo, Norway

**Comment by rungitta** [comment]
Test report by @rungitta
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
freebio2.hpc.uio.no - Linux RHEL 8.4, AArch64, ARM Cortex-A72 (graviton), Python 3.6.8
See https://gist.github.com/cb66e99381e44e77e052964141fa804b for a full test report.

File: easybuild/easyconfigs/j/Julia/Julia-1.6.0-linux-aarch64.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+toolchain = SYSTEM
+
+source_urls = ['https://julialang-s3.julialang.org/bin/linux/aarch64/%(version_major_minor)s/']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['0f496972d26cea88151204d03e6bd87702aa1ff983de3b1e4f320c48ef67325f']
+
+sanity_check_paths = {
+    'files': ['bin/julia', 'include/julia/julia.h', 'lib/libjulia.so'],
+    'dirs': ['bin', 'etc', 'include', 'lib', 'share']
+}
```
Comment (boegel) [inline easybuild/easyconfigs/j/Julia/Julia-1.6.0-linux-aarch64.eb:23]:
```suggestion
sanity_check_paths = {
    'files': ['bin/julia', 'include/julia/julia.h', 'lib/libjulia.%s' % SHLIB_EXT],
    'dirs': ['bin', 'etc', 'include', 'lib', 'share']
}

sanity_check_commands = ['julia --help']

modextravars = {
    # Use default DEPOT_PATH where first entry is user's depot
    # JULIA_DEPOT_PATH must be set to be able to load other JuliaPackage modules and have
    # those installations appended to DEPOT_PATH without disabling any of the default paths
    'JULIA_DEPOT_PATH': ':',
}
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — APPROVED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
aarch64-neoverse-v1-node1.int.aws-rocky88-202310.eessi.io - Linux Rocky Linux 8.8, AArch64, ARM UNKNOWN (neoverse_n1), Python 3.6.8
See https://gist.github.com/boegel/6bf9e3dd11546b7402d22beb7328acaf for a full test report.

**Comment by boegel** [comment]
Going in, thanks @rungitta!

**Pull request merged**